tiled/adapters/sql.py

```diff
@@ -305,8 +305,9 @@ def read(self, fields: Optional[List[str]] = None) -> pandas.DataFrame:
         Returns the concatenated pyarrow table as pandas dataframe.
         """
 
+        cols = ", ".join([f'"{c}"' for c in fields]) if fields else "*"
         query = (
-            f"SELECT * FROM {self.table_name} "
+            f'SELECT {cols} FROM "{self.table_name}" '
```

Comment:
:+1: :+1: :+1:

Do we need to take any additional precautions against SQL injection here? I've grown comfortable relying on frameworks for this and putting user-defined variables directly into SQL strings is scary.

Maybe just check that each `c` in `fields` cannot have any quotes in it? Better if there were some standard function in sqlalchemy or elsewhere that we could use.